### PR TITLE
[Bug] Fix prometheus read format error #13831

### LINKED ISSUE #13831: [Bug] Prometheus read format error
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.

### Version

9691db791807419e9cd017680e286c6fae65a68d

### What's Wrong?

introduced by #11693
Prometheus cannot read FE metrics because the format has error.
image
`# TYPE doris_fe_query_latency_ms_db=xxxx summary`

### What You Expected?

fix the histogram metrics format

### How to Reproduce?

_No response_

### Anything Else?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by ccoffline**
# Proposed changes

Issue Number: close #13831

## Problem summary

fix prometheus format error

## Checklist(Required)

1. Does it affect the original behavior: 
    - [x] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [x] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [x] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

### CODE REVIEW

**Comment by stalary** [comment]
Can you add some test?

**Comment by ccoffline** [comment]
> Can you add some test?

This bug is found because prometheus reports an error. So the only way to test this is to deploy Doris and let Prometheus tell us if the metrics api has an error.
I'll try to add some unit tests though.

**Comment by ccoffline** [comment]
> Can you add some test?

done

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.25 seconds
 load time: 432 seconds
 storage size: 17171500889 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20221114130322_clickbench_pr_45677.html

**Comment by spaces-X** [comment]
used to compatible with the metric group by db.

**Comment by spaces-X** [comment]
To compatible with the metric grouped by the database name.

File: fe/fe-core/src/main/java/org/apache/doris/metric/MetricRepo.java [hunk 1]
```diff
@@ -57,7 +57,7 @@ public final class MetricRepo {
     private static final Logger LOG = LogManager.getLogger(MetricRepo.class);
 
     // METRIC_REGISTER is only used for histogram metrics
-    private static final MetricRegistry METRIC_REGISTER = new MetricRegistry();
+    public static final MetricRegistry METRIC_REGISTER = new MetricRegistry();
```
Comment (spaces-X) [inline fe/fe-core/src/main/java/org/apache/doris/metric/MetricRepo.java:60]: for UT？

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**